### ISSUE TITLE
Setup for new Nuget based CMS project fails

### ISSUE THREAD

**Issue by simon-d** [OrchardCMS/OrchardCore#2551]
Currently it seems the process of setting up a new project based on the CMS targets nuget packages is borken. You are unable to access the Admin section of a new orchard core site setup using the OrchardCore.Application.Cms.Targets nuget package.

**Steps to reproduce:**
1) Setup new asp.net core site with 'dotnet new web'
2) Install latest Beta3 CMS Targets package from the myget feed
3) Register orchard core in the application Startup.cs
4) Run the application, and complete the Default tenant setup
5) Attempting to access /admin on the new site returns an internal server error

>  InvalidOperationException: Unable to resolve service for type 'OrchardCore.Email.ISmtpService' while attempting to activate 'OrchardCore.Email.Controllers.AdminController'.

**Comment by psijkof** [comment]
I believe this is related
https://github.com/OrchardCMS/OrchardCore/issues/2455

**Comment by psijkof** [comment]
Using VS 2017 15.8.7
Create new project -> ASP.NET Core web app -> empty
Adding OrchardCore.Application.Cms.Targets from https://www.myget.org/F/orchardcore-preview/api/v3/index.json, latest version (today this is 1.0.0-beta3-69124)
Changing my Startup.cs like so:
```
        public void ConfigureServices(IServiceCollection services)
        {
            services.AddOrchardCms();
        }

        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
        public void Configure(IApplicationBuilder app, IHostingEnvironment env)
        {
            if (env.IsDevelopment())
            {
                app.UseDeveloperExceptionPage();
            }

            app.UseOrchardCore();
        }
```
Running the project and setting up the default tenant, with SaaS recipe and SQLLite
Open /admin … works just fine in my case.
IOW I cannot reproduce what you see ...

**Comment by simon-d** [comment]
Ok I think found where my issue was, I had registered both orchard core CMS and MVC in ConfigureServices.

`public void ConfigureServices(IServiceCollection services)
        {
            services.AddOrchardCms().AddMvc();
        }`

I see now that Orchard is already registering MVC inside the AddOrchardCms() call. I did a quick test, adding the extra MVC service registration to a working site caused the error to appear again.

Looks like the documentation needs to be updated, as it shows adding the additional MVC service registration under Additional Frameworks here: https://orchardcore.readthedocs.io/en/latest/OrchardCore/OrchardCore.Modules/README/
